src/main/java/com/manardenza/dao/UserDaoImpl.java

```diff
@@ -3,6 +3,7 @@
 import com.manardenza.entity.User;
 
 import java.util.List;
+import java.util.Optional;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Optional'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)